python/sbp/signal.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+
+"""
+Struct to represent the signal(constellation, band, PRN).
+"""
+
+from construct import *
+import json
+from sbp.msg import SBP, SENDER_ID
+from sbp.utils import fmt_repr, exclude_fields, walk_json_dict, containerize, greedy_string
+
+# Automatically generated from piksi/yaml/swiftnav/sbp/signal.yaml with generate.py.
+# Please do not hand edit!
+
+
+class Signal(object):
+  """Signal.
+  
+  A struct to represent the signal(constellation, band, PRN).
```

Comment:
nit: "A struct to represent" => "Signal identifier containing constellation, band, and PRN." or something.